src/EVM/UnitTest.hs

```diff
@@ -257,7 +262,14 @@ symRun opts@UnitTestOptions{..} vm (Sig testName types) = do
     liftIO $ putStr txtResult
     liftIO $ printWarnings ends results t
     pure (not (any isCex results), not (warnings || unexpectedAllRevert))
-
+    where
+      symbolicFail :: Expr Buf -> Prop
+      symbolicFail e =
+        let text = V.fromList $ map (fromIntegral . ord) "assertion failed"
+            panic = e == ConcreteBuf (panicMsg 0x01)
+            assertFail =  V.take (length text) (Expr.concretePrefix e) == text
```

Comment:
I don't think this is sound? What if we have a symbolic prefix that could still be "assertion failed"?

Reply:
are we even able to test this case? we probably need to use assembly actually and like pass some symbolic memory region into revert directly?